ci/smoke_test.sh

```diff
@@ -1,7 +1,6 @@
 #!/usr/bin/env bash
 set -e
 
-apt-get update
-apt-get install curl
+apt-get --assume-yes update && apt-get --assume-yes install curl
```

Comment:
Since all we need is curl, we could change the task to use a lightweight image that comes built with curl then we wouldn't need to run this every build.